internal/python/plan.go

```diff
@@ -93,6 +93,13 @@ func DeterminePackageManager(ctx *pythonPlanContext) types.PackageManager {
 
 	for _, depFile := range depFiles {
 		if utils.HasFile(src, depFile.filename) {
+			if depFile.packageManagerID == types.PythonPackageManagerPdm {
+				if weakHasStringsInFile(src, []string{"pyproject.toml"}, "[tool.poetry]") {
```

Comment:
PDM has a field: `[tool.pdm]`. You can determine if a project should be installed with PDM by checking if this key group exists.

Reply:
A a simple pdm project, [tool.pdm] may not be included in pyproject.toml